libs/build_mapl.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -eux
+
+name="mapl"
+repo=${1:-${STACK_mapl_repo:-"GEOS-ESM"}}
+version=${2:-${STACK_mapl_version:-"main"}}
+
+# Hyphenated version used for install prefix
+compiler=$(echo $HPC_COMPILER | sed 's/\//-/g')
+mpi=$(echo $HPC_MPI | sed 's/\//-/g')
+id=${version//\//-}
+version_install=$repo-$id
+
+if $MODULES; then
+  set +x
+  source $MODULESHOME/init/bash
+  module load hpc-$HPC_COMPILER
+  module load hpc-$HPC_MPI
+  module load hpc-$HPC_PYTHON
+  module try-load cmake
+  module load esma_cmake
+  module load cmakemodules
+  module load ecbuild
+  module load gftl-shared
+  module load yafyaml
+  module load esmf
+  module load netcdf
```

Comment:
this order should be reversed.

Reply:
Addressed in 2c049520a84a6938bc1724058b4e2d81c30376ae